ui/packages/catalog-ui-search/src/main/webapp/react-component/container/results-export/results-export.tsx

```diff
@@ -49,36 +48,13 @@ class ResultsExport extends React.Component<Props, State> {
       selectedFormat: 'Select an export option',
       exportFormats: [],
       downloadDisabled: true,
-      ...this.mapSelectionToState(),
     }
-    this.props.listenTo(
```

Comment:
You can try verifying with @andrewkfiedler that removing this listener doesn't bring in any regressions

Reply:
There should not be any regressions in functionality since this component no longer depends on `store`.